pylint/lint/pylinter.py

```diff
@@ -709,7 +710,15 @@ def _get_asts(
         """Get the AST for all given FileItems."""
         ast_per_fileitem: dict[FileItem, nodes.Module | None] = {}
 
+        # For progress reports, it would be nice to say "item 1 of N",
+        # but we can't get the total count of filenames without
+        # materializing the list, or creating a second iterator and
+        # iterating.
+        progress_reporter = get_progress_reporter(self.verbose)
+        progress_reporter.print_message("Get ASTs.")
+
         for fileitem in fileitems:
+            progress_reporter.increment(fileitem.filepath)
```

Comment:
```suggestion
            progress_reporter.report_file(fileitem.filepath)
```

Perhaps? `increment` feels a bit weird.

Reply:
Ah right, I was using "increment" in the sense that we had a number of things to do, and the filename was just something that was being displayed.  "report_file" does feel more correct in this context.

Reply:
Fixed, is now `progress_reporter.report_file(fileitem.filepath)`